### PR TITLE
osu! Technical Championship #3: Wiki page

### PR DESCRIPTION
**Pull request by fajar13k**
- Add OTC #3 to Tournaments
- Add GS Match Results (First week)
- Add GS Mappools
- Label as blocked until 01/21

---

### CODE REVIEW

File: wiki/Tournaments/OTC/3/en.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+- 'OTC #3'
+- OTC 3
+- OTC#3
+- OTC3
+---
+# osu! Technical Championship #3
+
+![OTC #3 Logo](img/logo.png)
+
+The **osu! Technical Championship #3** (***OTC #3***) is a double elimination one-on-one osu! tournament hosted by ![][flag_US] [IsoSubjectPi](/users/3989669). osu! Technical Championship features complex rhythms and sliders, unorthodox patterns, and a wide variety of technical mapping in its mappools. It is the 3rd installment of the osu! Technical Championship.
```
Comment (cl8n) [inline wiki/Tournaments/OTC/3/en.md]:
```suggestion
The **osu! Technical Championship #3** (***OTC #3***) is a double-elimination one-on-one osu! tournament hosted by ![][flag_US] [IsoSubjectPi](/users/3989669). osu! Technical Championship features complex rhythms and sliders, unorthodox patterns, and a wide variety of technical mapping in its mappools. It is the third installment of the osu! Technical Championship.
```
same as last time :wink: i should probably change the other tournament articles sometime...

File: wiki/Tournaments/OTC/3/en.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+| Designers | ![][flag_DE] [Diffractive](/users/7601761), ![][flag_US] [DigitalHypno](/users/4384207) |
+| Streamers | ![][flag_US] [IsoSubjectPi](/users/3989669), ![][flag_US] [dblade](/users/4272841), ![][flag_DE] [Diffractive](/users/7601761), ![][flag_DE] [Ami](/users/2114779) |
+| Commentators | ![][flag_DE] [Diffractive](/users/7601761), ![][flag_SG] [woahsia](/users/195946), ![][flag_NL] [Blue-Orb](/users/6612210), ![][flag_US] [Woey](/users/3792472), ![][flag_DE] [Ami](/users/2114779), ![][flag_US] [itzpigeon](/users/9828300) |
+| Referees | ![][flag_US] [IsoSubjectPi](/users/3989669), ![][flag_DE] [Diffractive](/users/7601761), ![][flag_TR] [112servis](/users/3953470), , ![][flag_TW] [EthanTC](/users/9539163), ![][flag_FR] [Kasumii-sama](/users/6177263), ![][flag_FR] [\[ Mimir \]](/users/7382734), ![][flag_GB] [Nega](/users/3181083), ![][flag_DK] [Pennek](/users/3723742), ![][flag_DE] [Rymus](/users/6449966), ![][flag_US] [tigereyes144](/users/6499811), ![][flag_CN] [VRainbow](/users/3635214), ![][flag_BE] [DraQeu](/users/4425520), ![][flag_US] [Hot Cocoa](/users/4986705) |
```
Comment (cl8n) [inline wiki/Tournaments/OTC/3/en.md]:
```suggestion
| Referees | ![][flag_US] [IsoSubjectPi](/users/3989669), ![][flag_DE] [Diffractive](/users/7601761), ![][flag_TR] [112servis](/users/3953470), ![][flag_TW] [EthanTC](/users/9539163), ![][flag_FR] [Kasumii-sama](/users/6177263), ![][flag_FR] [\[ Mimir \]](/users/7382734), ![][flag_GB] [Nega](/users/3181083), ![][flag_DK] [Pennek](/users/3723742), ![][flag_DE] [Rymus](/users/6449966), ![][flag_US] [tigereyes144](/users/6499811), ![][flag_CN] [VRainbow](/users/3635214), ![][flag_BE] [DraQeu](/users/4425520), ![][flag_US] [Hot Cocoa](/users/4986705) |
```

**Comment by fajar13k** [comment]
> 
> 
> add this one to the main tournament index as well (for some reason it's already listed in the OTC index, spooky)

We can include it when we don't have the winners? Just asking more.

**Comment by WalterToro** [comment]
Two big notes here:
- Ro32 and Ro16 are on the same weekend but the schedule is only for Ro32, any reason to not add the tentative schedule for Ro16?, (probably could be too big of a table)
- Schedules are presented better chronologically, doesnt look too good when timestamps are kind of random, spectators could feel lost if they not know the order of the matches.
Even if the table is ordered by match ID, I need to be serious and say that not too many people use that in schedules, and its more of a brackets thing,

**Comment by fajar13k** [comment]
> Ro32 and Ro16 are on the same weekend but the schedule is only for Ro32, any reason to not add the tentative schedule for Ro16?, (probably could be too big of a table)

Sheet only has RO32 schedules, rest of 'em (which are RO16) are still "TBA", yet i don't know if the statiscian/host will tell us before weekend or not.

> Schedules are presented better chronologically, doesnt look too good when timestamps are kind of random, spectators could feel lost if they not know the order of the matches.
Even if the table is ordered by match ID, I need to be serious and say that not too many people use that in schedules, and its more of a brackets thing,

Will implement new one.

**Comment by cl8n** [comment]
I think it's fine to leave out ro16 schedule if it won't be provided until the weekend. and, yeah I think it's fine to add this tournament to the listing, write "*Ongoing*" in place of the winners maybe? or leave the boxes empty

**Comment by fajar13k** [comment]
Allright ready for more review

**Pull request merged**